generator/flask/generate.mjs

```diff
@@ -0,0 +1,132 @@
+import fs from 'fs/promises'
+import { dirname, join, basename } from 'path'
+import url from 'url'
+import mkdirp from 'mkdirp'
+import rimraf from 'rimraf'
+import { generateAppInformation } from '../generateAppInformation.mjs'
+
+const filesToIgnore = ['.next', '.env', 'generator-config.json', '.gitkeep', 'keys', 'appInformation.json', '__pycache__']
+const pathsToOverwrite = []
+
+const __dirname = dirname(url.fileURLToPath(import.meta.url))
+
+async function generate() {
+  await generateAppInformation()
+  const rootPath = join(__dirname, '../..')
+  const samplesPath = join(rootPath, 'samples')
+  const generatorPath = join(rootPath, 'generator')
+  const flaskPath = join(generatorPath, 'flask')
+  const overridesPath = join(flaskPath, 'overrides')
+  const templatePath = join(flaskPath, 'template')
+
+  const overrides = (await fs.readdir(overridesPath, { withFileTypes: true }))
+    .filter((i) => i.isDirectory())
+    .map((i) => i.name)
+    .sort()
+
+  console.log(`Detected samples: ${overrides.join(', ')}`)
+
+  for (const [i, sample] of overrides.entries()) {
+    console.log(`\nGenerating "${sample}" sample`)
+    const port = 3000 + i + 1
```

Comment:
Same for here, we can change this port value or remove if it is not being used

Reply:
done, removed.